src/utilities/index.js

```diff
@@ -22,3 +24,44 @@ export const stringToSearchRegExp = (value) => `/.*${value}.*/i`;
  */
 export const getOrganizationById = (organizations, organizationId) =>
   organizations.find(({ id }) => id === organizationId);
+
+/**
+ * @function
+ * @name covertDateStringToHumanReadableFormat
+ * @description receives a date iso string and converts it to a human readable format
+ *
+ * @param {string} dateString - date iso string to convert
+ * @param {string} format - format to convert to
+ * @link https://github.com/felixge/node-dateformat - library used to convert date(follow link to see format options)
+ *
+ * @returns {string} human readable date
+ */
+export const covertDateStringToHumanReadableFormat = (
```

Comment:
I'd suggest using or expanding the existing utility functions in `src/common/locale.js` to achieve this.
That way, the user will always see a format appropriate to their locale.